engine/transaction/draft/create_paymail_draft_test.go

```diff
@@ -0,0 +1,264 @@
+package draft_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/bitcoin-sv/go-paymail"
+	sdk "github.com/bitcoin-sv/go-sdk/transaction"
+	"github.com/bitcoin-sv/spv-wallet/engine/tester"
+	"github.com/bitcoin-sv/spv-wallet/engine/tester/paymailmock"
+	"github.com/bitcoin-sv/spv-wallet/engine/transaction"
+	"github.com/bitcoin-sv/spv-wallet/engine/transaction/draft"
+	"github.com/bitcoin-sv/spv-wallet/engine/transaction/draft/outputs"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreatePaymailDraft(t *testing.T) {
```

Comment:
Will it be beneficial to test some edge cases:
- transaction with zero satoshis
- transaction with extremely large values

Reply:
yes it will, thank you

Reply:
added tests for zero and negative satoshis and also for too large amount on JSON side